google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugProcessStateController.java

```diff
@@ -41,6 +43,8 @@
   private static final int INITIAL_DELAY_MS = 2000;
   private static final Logger LOG = Logger.getInstance(CloudDebugProcessStateController.class);
   private static final int PERIOD_MS = 500;
+  public static final String CLIENT_VERSION = "google.com\\/intellij\\/v" + ServiceManager
```

Comment:
Feel free to add a new method to the CloudToolsPluginInfoService

On Wed, Mar 30, 2016 at 12:26 PM, meltsufin <notifications@github.com>
wrote:

> In
> google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugProcessStateController.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/561#discussion_r57919092>
> :
>
> > @@ -41,6 +43,8 @@
> >    private static final int INITIAL_DELAY_MS = 2000;
> >    private static final Logger LOG = Logger.getInstance(CloudDebugProcessStateController.class);
> >    private static final int PERIOD_MS = 500;
> > +  public static final String CLIENT_VERSION = "google.com\\/intellij\\/v" + ServiceManager
>
> I didn't want to duplicate the prefix everywhere.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/561/files/56f05c74ef6ed99b523e00f9e4a884ac26465d88#r57919092>
>